### DISCUSSION TITLE
Has support for UWP been removed?

### DISCUSSION THREAD

**Discussion by HopefulFrog** [MonoGame/MonoGame#8187] [General]
I downloaded the Visual Studio extension for MonoGame templates, and I'm not seeing a UWP template. There's a "MonoGame Windows Universal XAML Application" template, which is tagged with UWP, but it's unclear whether that's the same thing. I'm assuming that it is, but the title is confusing if so.

I want to put my game on Xbox and the Windows Store using UWP. Is that the template to use?

**Comment by ThomasFOG** [comment]
```MonoGame Windows Universal XAML Application``` is what you are looking for ~~but...~~

~~... since January 1st, it is no longer possible to submit an UWP app to the Windows Store. So UWP is basically useless now and only around for people submitting patches of already published apps.~~

UWP support is going to be removed from the next release. It'll be replaced by a new target based upon the GDK platform (there is no ETA for that for the time being, but the Foundation is going to issue a bounty to accelerate it in the upcoming days).

**Reply by HopefulFrog** [reply]
That's unfortunate. I tried searching for this but couldn't find any information or articles about UWP being blocked from the Store starting this year. Do you have a link where I could read more about it?

Also, what about [this post on the Discourse forums](https://community.monogame.net/t/has-microsoft-ended-the-xbox-creators-program/19811/5) from an Xbox team member late last year?

**Reply by ThomasFOG** [reply]
Hmmm, it would appear that this date is basically an urban legend going around following a store bug that is now resolved. I'll try to get a solid answer.

That being said, once the GDK implementation will be there, MonoGame will remove support for UWP.

**Reply by HopefulFrog** [reply]
Understood. So when that happens, if I want to use UWP, I'll have to remain on prior versions of MonoGame that do support it.

**Reply by ThomasFOG** [reply]
Yes.
